internal/jstest/jstest.go

```diff
@@ -75,7 +73,10 @@ func Run(ctx context.Context, dir string, args []string) (*internal.TestResults,
 			it := &item{
 				file: f,
 			}
-			it.out, it.err = runCommand(dir, "mongo", filepath.Join(volume, f))
+
+			// we set working_dir so a relative path is needed here
+			rel, _ := filepath.Rel("mongo", f)
```

Comment:
I purposefully ignored an error here as I cannot see under what circumstances an error would occur.

Should I return one anyway?